index.html

```diff
@@ -1,29 +1,230 @@
+  </head>
+  <body>
+    <header class="up-side">
+      <p class="logo">My Logo</p>
+      <nav class="union">
+        <img src="icons/Union.png" />
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="alex">
+        Hey There. <br />
+        I'm Alex.
+      </h1>
+      <h3 class="soft">I'm a Software Developer</h3>
+      <p class="text">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me.
+      </p>
+      <ul class="icon">
+        <li class="icon1"><img src="icons/icon1.png" /></li>
+        <li class="icon2"><img src="icons/icon2.png" /></li>
+        <li class="icon3"><img src="icons/icon3.png" /></li>
+        <li class="icon4"><img src="icons/icon4.png" /></li>
+        <li class="icon5"><img src="icons/icon5.png" /></li>
+      </ul>
     </section>
-</body> 
-</html>
\ No newline at end of file
+    <section class="work">
+      <h3 class="recent-works">My Recent Works</h3>
+      <div class="vector">
+        <img src="icons/vector.png" />
+      </div>
+      <div class="image">
+        <img src="icons/image1.png" class="background-img" />
+      </div>
+      <div class="work-text">
+        <h4 class="work-title">Multi Post Stories</h4>
+        <p class="inside-text">
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required. has been the industry's standard dummy text ever
+          since the 1500s, when an unknown printer took a standard dummy text.
+        </p>
+        <ul class="languages">
+          <button class="lang"><li>CSS</li></button>
+          <button class="lang"><li>HTML</li></button>
+          <button class="lang"><li>Bootstrap</li></button>
+          <button class="lang"><li>Ruby</li></button>
+        </ul>
+      </div>
+      <ul>
+        <button class="see-project">
+          <li>See Project</li>
+        </button>
+      </ul>
+      <div class="background">
+        <img src="icons/background.png" class="background-img" />
+        <div class="back-button">
+          <button class="button1">See Project</button>
+        </div>
+      </div>
+      <ul class="container">
+        <li class="grid-container">
+          <div class="work-text1">Profesional Art Printing Data</div>
+          <div class="work-text2">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. Has been the industry's standard.
+          </div>
+
+          <ul class="new-lang">
+            <button class="last-lang"><li>HTML</li></button>
+            <button class="last-lang"><li>Bootstrap</li></button>
+            <button class="last-lang"><li>Ruby</li></button>
+          </ul>
+        </li>
+      </ul>
+      <div class="background">
+        <img src="icons/background.png" class="background-img" />
```

Comment:
- [ ] Kindly as requested by the previous reviewer Please make sure that background images are implemented as background images and not as imgs on the page. You can use the [CSS background-position Property](https://www.w3schools.com/cssref/pr_background-position.php) to properly position the image in the right place.
- NB: and consider doing that on all projects cards to meet the below requirement
> The section has the proper background image (decorations are always backgrounded images, not img tags)